### PR TITLE
Add summaries to summer-2022 and summer-2021 videos

### LINKED ISSUE #33: Add summaries for You Got This Summer 2021 videos
[Add summaries](https://github.com/yougotthisconf/yougotthis.io/blob/main/.github/CONTRIBUTING.md#content-summaries) to all videos in `/content/library/videos/summer-2021`

### LINKED ISSUE #34: Add summaries for You Got This Summer 2022 videos
[Add summaries](https://github.com/yougotthisconf/yougotthis.io/blob/main/.github/CONTRIBUTING.md#content-summaries) to all videos in `/content/library/videos/summer-2022`

### PR DESCRIPTION
**Pull request by harshitajain165**
Closes #33 and closes #34

### CODE REVIEW

File: content/library/videos/summer-2021/align-values-with-job.md [hunk 1]
```diff
@@ -20,6 +20,23 @@ duration: 25
 cover: >-
   https://i.vimeocdn.com/video/1349709776-7f64dd2de2dc22bee68b7b241346ad8edfe5514c2ca2454e6ed3271ef4e8ce81-d
 ---
+<extra-reading title="Summary">
+<!-- There is am empty line required above and below content. -->
+
+While motivation and acquiring discipline are important to do our jobs well, the secret to remaining aligned with what we do is by keeping the inner child spirit alive.
+- Step 1: Know your values
```
Comment (phazonoverload) [inline content/library/videos/summer-2021/align-values-with-job.md:27]: The formatting of these looks weird. Can you make it one line with the content, and use a `<br>` tag to break line?

File: content/library/videos/summer-2021/align-values-with-job.md [hunk 2]
```diff
@@ -20,6 +20,23 @@ duration: 25
...
+  
+This requires a lot of self-awareness. For people just starting their careers, Laura suggests that they should try as many things and jobs as they can. Trying new things will help them learn their likes and dislikes.
+- Step 2: Check your emotions
```
Comment (phazonoverload) [inline content/library/videos/summer-2021/align-values-with-job.md:29]: Same here

File: content/library/videos/summer-2021/civil-engineering-ethics.md [hunk 3]
```diff
@@ -24,6 +24,21 @@ duration: 24
 cover: >-
   https://i.vimeocdn.com/video/1349710007-b3a5645d6cdea2049bd30e8a9a9954cbed2cef992e786731f7f4c0d245669dbf-d
 ---
+<extra-reading title="Summary">
+<!-- There is am empty line required above and below content. -->
+
+Hundred years ago, civil engineering was very similar to how the tech industry is today. As the industrial revolution receded behind engineers, they found new ways to use the technologies they had developed. But as these projects became more ambitious, there was a rise in major engineering disasters due to questionable ethics. These disasters forced engineers to confront their shortcomings. As a result, they began to regulate themselves more intensely in established standardised industry codes of ethics.
```
Comment (phazonoverload) [inline content/library/videos/summer-2021/civil-engineering-ethics.md]:
```suggestion
A hundred years ago, civil engineering was very similar to how the tech industry is today. As the industrial revolution receded behind engineers, they found new ways to use the technologies they had developed. But as these projects became more ambitious, there was a rise in major engineering disasters due to questionable ethics. These disasters forced engineers to confront their shortcomings. As a result, they began to regulate themselves more intensely in established standardised industry codes of ethics. 
```

File: content/library/videos/summer-2022/overcoming-self-sabotage.md [hunk 4]
```diff
@@ -21,6 +21,17 @@ cover: >-
...
+<!-- There is am empty line required above and below content. -->
+
+The reason why we sabotage ourselves is that our current habits are familiar and comfortable while the change we desire is inherently unpredictable. We may think that self-sabotage comes from not knowing what we want but we don’t sabotage knowledge or desire, we sabotage the transformation. Our sabotaging behaviours operate in the subconscious which gives them power while our transforming behaviours are weaker and require our conscious awareness and choice. 
+
+How to overcome sabotage?
+  
+We must pull the sabotaging behaviour into our conscious mind i.e. catch them in their pre-state. Although we still might choose sabotaging behaviour, the victory lies in making that behaviour deliberate. As we get better at catching the sabotaging behaviour, we can then exercise our decisive powers to choose the new behaviour. Short-circuiting sabotaging bearings has a waterfall effect. Some sabotaging behaviours are connected; getting a handle on one leads to control over the others. Eventually, sabotage recognition gets easier and our new behaviours have a chance to establish in our subconscious.
```
Comment (phazonoverload) [inline content/library/videos/summer-2022/overcoming-self-sabotage.md]:
```suggestion
To overcome sabotage, we must pull the sabotaging behaviour into our conscious mind i.e. catch them in their pre-state. Although we still might choose sabotaging behaviour, the victory lies in making that behaviour deliberate. As we get better at catching the sabotaging behaviour, we can then exercise our decisive powers to choose the new behaviour. Short-circuiting sabotaging bearings has a waterfall effect. Some sabotaging behaviours are connected; getting a handle on one leads to control over the others. Eventually, sabotage recognition gets easier and our new behaviours have a chance to establish in our subconscious.
```

**Review by phazonoverload — CHANGES_REQUESTED** [review]
Really fantastic contribution, thank you! I've suggested a couple of small changes for formatting and conciseness.

**Comment by harshitajain165** [comment]
Thank you @phazonoverload, I've made the suggested changes.

**Review by phazonoverload — APPROVED** [review]

**Pull request merged**